dunai-frp-bearriver/src/FRP/BearRiver.hs

```diff
@@ -436,6 +437,12 @@ dSwitch sf sfC = MSF $ \a -> do
                        return (b, ct')
     (b, NoEvent) -> return (b, dSwitch ct sfC)
 
+link :: Monad m => SF m a (Either b c) -> (c -> SF m a (Either b d)) -> SF m a (Either b d)
```

Comment:
This should have a haddock.

Reply:
Done. I added them to `switch` and `dSwitch` as well.